packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,23 @@
+## 46.11.0 - August 22, 2019
+
+## Fixed
+
+#### salesforcedx-core
+
+- Add CustomObjects as a bundle type for retrieval with the Org Browser ([PR #1569](https://github.com/forcedotcom/salesforcedx-vscode/pull/1569))
+
+- Fix retrieval of standard objects for the Org Browser ([PR #1571](https://github.com/forcedotcom/salesforcedx-vscode/pull/1571))
```

Comment:
I would include this under Added, what do you think @brpowell @AnanyaJha

Reply:
Agreed, let's move this one to Added because we weren't displaying standard objects at all previously